### PR TITLE
Feature/gulp workflow

### PR DESCRIPTION
**Pull request by geoffgraham**

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
+var gulp             = require('gulp'),
+    plumber          = require('gulp-plumber'),
```
Comment (barryhughes) [inline gulpfile.js]: Preferred:

```js
var abc = require('module-1');
var def = require('module-2');
```

(Less churn during future changesets/reviews.)

File: gulpfile.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    livereload       = require('gulp-livereload');
+
+/**
+ * Compile styles
+ */
+gulp.task('styles', function() {
+	return sass( 'scss/*' )
+		.pipe(autoprefixer())
+    .pipe(cleanCSS())
+    .pipe(rename('style.min.css'))
```
Comment (barryhughes) [inline gulpfile.js]: Rule of thumb: initial indent should be tabs, subsequent indenting should be spaces. (We have spaces here and tabs on other lines.)

File: package.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
+{
+	"name": "foursquare-org",
```
Comment (barryhughes) [inline package.json]: Can we clean this up? Ditto other references to foursquare.org and LimeCuda.

File: gulpfile.js (hunk 1 above)
Comment (geoffgraham) [reply gulpfile.js]: Cool, done!

File: gulpfile.js (hunk 2 above)
Comment (geoffgraham) [reply gulpfile.js]: Sorry dude, all set!

File: package.json (hunk 3 above)
Comment (geoffgraham) [reply package.json]: Wiped clean!

**Pull request merged**